src/Commands/Backup/Automatic/EnableCommand.php

```diff
@@ -33,6 +33,12 @@ class EnableCommand extends TerminusCommand implements SiteAwareInterface
     public function enableSchedule($site_env, $options = ['day' => null, 'keep-for' => null,])
     {
         list(, $env) = $this->getSiteEnv($site_env);
+
+        // Schedule Backup workflow expects 'weekly-ttl' as input option.
+        if (isset($options['keep-for']) && !is_null($options['keep-for'])) {
```

Comment:
```suggestion
        if (isset($options['keep-for'])) {
```

or consider this variant (if `null` on `$options['weekly-ttl']` acceptable:
```suggestion
        $options['weekly-ttl'] = $options['keep-for'] ?? null;
```